src/entities/LoginLogic.java

```diff
@@ -1,58 +1,100 @@
 package entities;
 
-import java.util.ArrayList;
+import java.sql.*;
 import java.util.List;
 
-public class LoginLogic{
-
-
-
+/**
+ * @author Mirko Di Cristina
+ * Class of Login Logic with SQL
+ */
+public class LoginLogic {
 
     List<User> listOfUser;
-
+    String name;
+    String surname;
+    String email;
+    String password;
+    int age;
+    Date birthday;
+    String gender;
 
 
     public LoginLogic() {
-        this.listOfUser = new ArrayList<>();
+
+    }
+
+    public LoginLogic(String name, String surname, String email, String password, int age, Date birthday, String gender) {
+        this.name = name;
+        this.surname = surname;
+        this.email = email;
+        this.password = password;
+        this.age = age;
+        this.birthday = birthday;
+        this.gender = gender;
     }
 
     /**
-     * @author Giuseppe Corrao
-     * @version 1.0
-     * login per User
+     * @author Mirko Di Cristina
+     * @param registeredUsers registration parameters
+     * @return User registration
      */
-    public void registered(String email,String password){
-        User newUser = new User(email,password);
-        this.listOfUser.add(newUser);
+    public boolean register(User registeredUsers) {
 
-        }
+        try {
 
-    public List<User> getListOfUser() {
-        return listOfUser;
-    }
+            Connection connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/newdb", "developer", "developer");
 
-    public void setListOfUser(List<User> listOfUser) {
-        this.listOfUser = listOfUser;
-    }
+            PreparedStatement pst = connection.prepareStatement(" INSERT INTO User (User_name, User_surname, User_age, User_birthday, User_gender, User_email, User_password) VALUES (?,?,?,?,?,?,?)");
+
+            pst.setString(1, registeredUsers.nameUser);
+            pst.setString(2, registeredUsers.surnameUser);
+            pst.setInt(3, registeredUsers.age);
+            pst.setDate(4, (Date) registeredUsers.birthday);
+            pst.setString(5, registeredUsers.gender);
+            pst.setString(6, registeredUsers.email);
+            pst.setString(7, registeredUsers.password);
+
+            pst.executeUpdate();
+            return true;
 
-    public String login(String email, String password) {
-        for (int i = 0; i < listOfUser.size(); i++) {
-            if (listOfUser.get(i).email.equals(email) && listOfUser.get(i).password.equals(password)) {
-                return "you are login";
-            }
 
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
         }
-        return "password and email are wrong";
+
+        return false;
```

Comment:
in questo modo la variabile booleana non sarà sempre false ?, false lo potrebbe restituire all'interno del catch